tableau_utilities/tableau_server/tableau_server.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, user, password, host, site, api_version=3.16):
         """
         self.user = user
         self.__pw = password
+        self.token_secret = token_secret
+        self.token_name = token_name
```

Comment:
```suggestion
        self.token_secret = personal_access_token_secret
        self.token_name = personal_access_token_name
```
Not sure if it matters too much, but it might be good for the explicitly call these `personal_access_token` to make it clear what kind of token details the user is expected to pass.

Reply:
Yeah, I will rename.